teps/0092-scheduling-timeout.md

```diff
@@ -45,8 +49,22 @@ authors:
 
 ## Summary
 
-Add runtime configuration options to allow setting a timeout for `TaskRuns` which applies before the `TaskRun's`
-underlying pod is scheduled, and only start our existing timeout counters after this timeout has passed, if specified.
+This TEP proposed adding runtime configuration options to allow setting a timeout for `TaskRuns` which would have 
+applied before the `TaskRun's` underlying `Pod` is scheduled, then we would start our existing timeout counters after
+this new timeout had passed, if specified.
```

Comment:
The use cases for this feature request were listed as:

* The user has N number of `pipelineTasks` running in parallel. This is very resource intensive and causing a huge load on a node. This load is causing other unrelated `taskRuns` (independent or part of a pipeline) to timeout without getting executed. How to save those unrelated `taskRuns` from timing out?

* Create extreme high number of `pipelineRuns` and `taskRuns` resulting in an overloaded cluster. These `pipelineRuns` could be a million runs of the same `pipeline` or  a combination of N different `pipelines`. These `pipelines` could be related or unrelated i.e. accessing the same remote resources. When a `node` is overloaded with such `fire and forget` nature of the operation, the expectation is to somehow maintain a `queue` of such `pipelineRuns`/`taskRuns` and cancel them when a new `pipelineRuns` are created. The pipeline controller has all the freedom to cancel any `pipelineRuns` from the `queue`.

* Align with GCP [queueTtl](https://cloud.google.com/build/docs/build-config-file-schema#queuettl)

Reply:
Yes, and those use cases are represented in TEP-0132 - https://github.com/tektoncd/community/pull/968/files#diff-2897584b231ee227ca79f2e20da5b4dc7f62c835f1c94a90046ef1c5bd4704ff

Didn't include the GCP queueTttl parity, I guess to remove vendor specificity - cc @lbernick